pandas/core/aggregation.py

```diff
@@ -196,3 +254,23 @@ def maybe_mangle_lambdas(agg_spec: Any) -> Any:
         mangled_aggspec = _managle_lambda_list(agg_spec)
 
     return mangled_aggspec
+
+
+def _relabel_result(result, reordered_result, func, columns, order):
+    """Internal function to reorder result if relabelling for dataframe.agg."""
+
+    reordered_indexes = [
+        pair[0] for pair in sorted(zip(columns, order), key=lambda t: t[1])
+    ]
+    idx = 0
+
+    # The reason is self._aggregate outputs different type of result if
+    # any column is only used once in aggregation
+    mask = isinstance(result, ABCDataFrame) and result.isna().any().any()
```

Comment:
Why do we need multiple `.any()` calls here?

Reply:
this is because `result.isna()` is still dataframe, and `.any().any()` will return bool, if only one `any`, will be a series and we dont want it for the mask

Reply:
I don’t totally understand this - can you provide an example?

Reply:
you can use np.any